packs/data/actions.db/barbed-quills.json

```diff
@@ -13,7 +13,7 @@
             "value": null
         },
         "description": {
-            "value": "<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You are hit with an unarmed strike or a strike with a non-reach melee weapon.</p>\n<hr />\n<p>You break off quills in your attacker's flesh. You deal [[/r 1d8[piercing]]] damage to the triggering creature (@Check[type:reflex|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true] save using your class DC or spell DC, whichever is higher.) On a critical failure, the creature also takes @Localize[PF2E.PersistentDamage.Bleed1d4.success] as your quills hook into its flesh. At 3rd level, and every 2 levels thereafter, this damage increases by 1d8 and the persistent piercing damage increases by 1.</p>\n<p>[[/r ceil(@actor.level/2)d8[piercing]]]{Leveled} piercing damage</p>\n<p>[[/r (1d4+ceil(@actor.level/2)-1)[bleed]]]{Leveled} bleed damage</p>\n<hr />\n<p><em>Note: +1 bonus from Bristle is not added automatically</em></p>"
+            "value": "<p><strong>Frequency</strong> once per day</p>\n<p><strong>Trigger</strong> You are hit with an unarmed strike or a strike with a non-reach melee weapon.</p>\n<hr />\n<p>You break off quills in your attacker's flesh. You deal [[/r 1d8[piercing]]] damage to the triggering creature (@Check[type:reflex|dc:resolve(@actor.attributes.classOrSpellDC.value)|basic:true] save using your class DC or spell DC, whichever is higher.) On a critical failure, the creature also takes [[/r 1d4[bleed]]] as your quills hook into its flesh. At 3rd level, and every 2 levels thereafter, this damage increases by 1d8 and the persistent piercing damage increases by 1.</p>\n<p>[[/r ceil(@actor.level/2)d8[piercing]]]{Leveled} piercing damage</p>\n<p>[[/r (1d4+ceil(@actor.level/2)-1)[bleed]]]{Leveled} bleed damage</p>\n<hr />\n<p><em>Note: +1 bonus from Bristle is not added automatically</em></p>"
```

Comment:
I think we need "damage" appended to these

Reply:
![image](https://user-images.githubusercontent.com/80183198/225116579-5d9b7396-bb0d-4179-93f6-80b3ac33883f.png)

It's not in the localization so text wise nothing would change.

Reply:
It is possible that many of these are missing the word after the roll though just in general. That's going to be an awful QA process.